app/assets/stylesheets/projects/index.scss

```diff
@@ -138,27 +133,39 @@ $demo-btn-shadow: #3C93D5;
       height: 57px;
       line-height: 57px;
       overflow: hidden;
-      padding: 0 8px;
+      color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.